app/services/dashboard/manager.rb

```diff
@@ -30,12 +31,47 @@ def find_default_widget_by_name(name)
 
       def builtin_widgets
         [
-          {template: 'status_widget',       sizex: 8, sizey: 1, name: N_('Host Configuration Status')},
-          {template: 'status_chart_widget', sizex: 4, sizey: 1, name: N_('Host Configuration Chart')},
-          {template: 'reports_widget',      sizex: 6, sizey: 1, name: N_('Latest Events')},
-          {template: 'distribution_widget', sizex: 6, sizey: 1, name: N_('Run Distribution Chart')},
-          {template: 'new_hosts_widget',    sizex: 8, sizey: 1, name: N_('New Hosts')}
-        ]
+          (registered_report_orgins + ['All']).sort.flat_map do |origin|
+            [
+              {
+                template: 'status_widget',
+                sizex: 8,
+                sizey: 1,
+                name: N_('Host Configuration Status for %s') % origin,
+                settings: {
+                  origin: origin,
+                  class_name: 'host-configuration-status'
+                }
+              },
+              {
+                template: 'distribution_widget',
+                sizex: 6,
+                sizey: 1,
+                name: N_('Run Distribution Chart for %s') % origin,
```

Comment:
for some reason I'm getting "Run Distribution Chart for" without type listed as labels for all 3 charts (all, puppet, ansible)